docusaurus/docs/create-a-plugin/develop-a-plugin/best-practices.md

```diff
@@ -0,0 +1,64 @@
+---
+id: best-practices
+title: Best practices
+description: An index of best practices for plugins.
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - best practices
+sidebar_position: 5
+---
+
+This document summarizes best practices for Grafana plugins. This live document is updated as new best practices are identified.
+
+## General
+
+- Choose the right plugin type for your use case: panel, data-source or app. See [Plugin types](/introduction/plugin-types-usage) for more information.
+- Verify that your data-source or app plugin can be provisioned. See [Provisioning](https://grafana.com/docs/grafana/latest/administration/provisioning/#data-sources) for more information.
+- Include default dashboards in your data-source or app plugin. See [Bundling of dashboards](https://grafana.com/docs/grafana/latest/administration/provisioning/#data-sources) for more information.
```

Comment:
Maybe we want to link to https://grafana.com/developers/plugin-tools/introduction/plugin-types-usage#bundling-of-dashboards?

Reply:
wrong link, good catch :+1: